packages/db/lib/migrator.mjs

```diff
@@ -148,6 +150,18 @@ class Migrator {
     return false
   }
 
+  checkIfMigrationFilesExist () {
+    try {
+      if (readdirSync(this.migrationDir).length === 0) {
```

Comment:
Could you use the promise-based version instead?

Reply:
Yes, I'll do that

Reply:
Done, @mcollina .

One question though. Why would we use the async version if we are having to wait for `setupPostgrator` and `postgrator.migrate` ?